lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -199,16 +200,29 @@ public void execute(Publisher publisher) throws ConnectorException {
         publisher.publish(doc);
       }
 
-      // close all results
-      rs.close();
-      for (ResultSet ors : otherResults) {
-        ors.close();
-      }
       // the post sql.
       runSql(connection, postSql);
       flush();

```

Comment:
The flush() method definition and this reference can be removed. flush() is a private method with an empty body -- it is not serving a purpose

Reply:
Done